src/lib/server/withClerkHandler.ts

```diff
@@ -43,6 +45,10 @@ export default function withClerkHandler(middlewareOptions?: ClerkSvelteKitMiddl
 
 		const authObject = requestState.toAuth()
 		event.locals.auth = authObject
+		sharedAuth.set(
+			makeAuthObjectSerializable(stripPrivateDataFromObject(authObject)) as unknown as InitialState
+		)
```

Comment:
Oh, we should not using shared stores/state in server. It's in the [docs](https://kit.svelte.dev/docs/state-management#avoid-shared-state-on-the-server)

Also see https://github.com/sveltejs/kit/discussions/4339#discussioncomment-6105163

Reply:
You are right... There is a issue on svelte to support isolated states: https://github.com/sveltejs/svelte/issues/13594